opportunity/examples/umbra/OpportunityRequestUmbra.json

```diff
@@ -0,0 +1,18 @@
+{
+  "product_id": "umbra_spotlight",
+  "datetime": "2024-04-19T00:00:00Z/2024-04-23T00:00:00Z",
+  "geometry": {
+    "type": "Point",
+    "coordinates": [13.403258555886767, 52.473696635108176]
+  },
+  "filter": {
+    "op": "and",
+    "args": [{
+      "op": ">=",
+      "args": ["grazingAngleDegrees", 45]
```

Comment:
```suggestion
      "args": [{"property": "grazingAngleDegrees"}, 45]
```

Reply:
This construct is for all properties that are supported according to cql2 construct.